projects/igniteui-angular/src/lib/carousel/carousel.component.ts

```diff
@@ -305,9 +369,9 @@ export class IgxCarouselComponent implements OnDestroy {
 
     /**
      * Transitions to the previous slide in the carousel.
-     *
-     * @returns
-     *
+     * ```typescript
+     * let previousSlide = this.carousel.prev();
```

Comment:
prev() and next() do not return a value, code snippet should be just this.carousel.prev()